File: easterEggs/bitterKola/TaxOwed.java
import java.util.Scanner;

public class TaxOwed{


public static void main (String [] args){



Scanner sc = new Scanner(System.in);

double taxCounter = 0



System.out.print("Enter the annual tax");

double tax1 = sc.nextDouble();


if(tax1>300000 && tax1<=600000){

 double tax1= tax1*0.15;

} else if (tax1 > 600000){

double tax1= tax1*0.25;
}

System.out.print(tax1);























}













}

File: Python eastereggs/bitterkola/taxowed.py

print("Enter the annual tax", end="")
tax1 = float(input())

if tax1 > 300000 and tax1 <= 600000:
    tax1 = tax1 * 0.15
elif tax1 > 600000:
    tax1 = tax1 * 0.25

print(tax1, end="")
